pkg/resources/prometheus/configmap.go

```diff
@@ -183,6 +183,9 @@ func renderTemplate(tpl string, data interface{}) (string, error) {
 	return strings.TrimSpace(output.String()), nil
 }
 
+// While updating the prometheusConfig template, remember that tab characters in the template (`\t`)
+// that might be used for indentation can cause syntax errors.
+// Please ensure to use spaces instead of tabs to prevent syntax errors in this template.
```

Comment:
This comment is weird. The content is YAML and YAML doesn't allow tabs for indentation anyway.